packages/compass-export-to-language/src/stores/store.js

```diff
@@ -57,37 +57,52 @@ const configureStore = (options = {}) => {
       'open-aggregation-export-to-language',
       (aggregation) => {
         store.dispatch(modalOpenChanged(true));
-        store.dispatch(inputExpressionChanged({ aggregation: aggregation }));
+        store.dispatch(
+          inputExpressionChanged({
+            aggregation: aggregation,
+            exportMode: 'Pipeline',
+          })
+        );
       }
     );
 
-    localAppRegistry.on('open-query-export-to-language', (queryStrings) => {
-      const query = {};
-      if (typeof queryStrings === 'string') {
-        query.filter = queryStrings === '' ? '{}' : queryStrings;
-      } else {
-        [
-          'filter',
-          'project',
-          'sort',
-          'collation',
-          'skip',
-          'limit',
-          'maxTimeMS',
-        ].forEach((k) => {
-          if (!queryStrings[k] || queryStrings[k] === '') {
-            if (k === 'filter') {
-              query[k] = '{}';
+    localAppRegistry.on(
+      'open-query-export-to-language',
+      (queryStrings, exportMode) => {
+        const query = {};
+        if (typeof queryStrings === 'string') {
+          query.filter = queryStrings === '' ? '{}' : queryStrings;
```

Comment:
So did you find any case in the codebase we're leaving this here for?

Reply:
Ah no, you are right, after refactoring forgot to remove this line of code. I'll remove it and merge with evergreen finises.